src/Okta.Sdk.UnitTests/DefaultRequestExecutorShould.cs

```diff
@@ -85,5 +85,66 @@ public async Task NotRetryRequestOnceWhenResponseIsNot401AndAuthorizationModeIsP
             tokenMessageHandler.NumberOfCalls.Should().Be(1);
             requestMessageHandler.NumberOfCalls.Should().Be(1);
         }
+
+        [Theory]
+        [InlineData(HttpVerb.Get)]
+        [InlineData(HttpVerb.Post)]
+        [InlineData(HttpVerb.Put)]
+        [InlineData(HttpVerb.Delete)]
+        public async Task ExecuteMethodBasedOnVerb(HttpVerb verb)
+        {
+            var httpClientRequest = new HttpClient();
+            var configuration = new OktaClientConfiguration
+            {
+                OktaDomain = "https://myOktaDomain.oktapreview.com",
+                AuthorizationMode = AuthorizationMode.PrivateKey,
```

Comment:
Define `Token = "foo"` so you can remove all oAuth code related and simplify this test.